tests/data_files/keyfile_1024

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Since you're renaming those files, could you please give them less generic names that convey what the file contains? Something like `keyfile_rsa_1024_cleartext.pem`, `keyfile_rsa_1024_3des.pem`, etc.

Reply:
@gilles-peskine-arm Thanks, I've unified the naming scheme for the RSA keys now.